index.js

```diff
@@ -898,16 +916,15 @@ class Client {
 
     async function getBulkLocation() {
       const response = await this.request("/api/v1/bulk-request", {
-        method: "POST",
-        body: requests,
+        method: "post",
+        data: requests,
         headers: {
           "content-type": "application/json"
-        },
-        simple: false,
-        resolveWithFullResponse: true
+        }
       });
-      if (response.statusCode === 303 && response.caseless.get("Location")) {
-        return response.caseless.get("Location");
+
+      if (response.statusCode === 200 && response.redirectCount > 0) {
+        return response.headers["content-location"];
```

Comment:
Is there documentation with respect to this change in spec?

Reply:
Axios cannot follow redirects, what it can do though is give us a count of the redirects it went through.

There is no doc for the same on their site

Reply:
https://axios-http.com/docs/req_config